### ISSUE TITLE
[Research] OUI Compliance audit of `src/core/public/core_app`

### ISSUE THREAD

**Issue by BSFishy** [opensearch-project/OpenSearch-Dashboards#4114]
Tracking issue for OUI compliance audit of the `src/core/public/core_app` folder.

Follow the steps in #3945 to complete the audit, then open another issue with findings. Once the work of performing the audit is done, this issue can be closed.

Note that this is a core service, which may be involved with the implementation and inclusion of OUI styles and themes. Since this is a core service, the conclusion of this audit might actually not involve any additional work.

<details>
  <summary>Related files</summary>

size (bytes) | file | notes
---|---|---
2.6K | [src/core/public/core_app/styles/_mixins.scss](https://github.com/opensearch-project/OpenSearch-Dashboards/tree/main/src/core/public/core_app/styles/_mixins.scss) | 
272 | [src/core/public/core_app/styles/_globals_v8light.scss](https://github.com/opensearch-project/OpenSearch-Dashboards/tree/main/src/core/public/core_app/styles/_globals_v8light.scss) | 
269 | [src/core/public/core_app/styles/_globals_v8dark.scss](https://github.com/opensearch-project/OpenSearch-Dashboards/tree/main/src/core/public/core_app/styles/_globals_v8dark.scss) | 
232 | [src/core/public/core_app/styles/_globals_v7light.scss](https://github.com/opensearch-project/OpenSearch-Dashboards/tree/main/src/core/public/core_app/styles/_globals_v7light.scss) | 
229 | [src/core/public/core_app/styles/_globals_v7dark.scss](https://github.com/opensearch-project/OpenSearch-Dashboards/tree/main/src/core/public/core_app/styles/_globals_v7dark.scss) | 

</details>

If this issue isn't already assigned to someone and you'd like to take on this work, please ping @BSFishy in the comments and I can assign it to you.
